packages/amplify-util-mock/src/api/api.ts

```diff
@@ -92,6 +110,17 @@ export class APITest {
       context.print.error(`Failed to stop DynamoDB Local Server ${e.message}`);
     }
 
+    try {
+      if (this.opensearchEmulator) {
+        await this.opensearchEmulator.terminate();
+        this.opensearchEmulator = null;
+        this.opensearchURL = null;
+      }
+    } catch (e) {
+      // failed to stop opensearch emulator
+      context.print.error(`Failed to stop OpenSearch Local Server ${e.message}`);
```

Comment:
What action should someone take if this happens?

Also, please use `printer.error` from `amplify-prompts`